src/Admin/AdministratorAdmin.php

```diff
@@ -73,6 +73,7 @@ protected function configureFormFields(FormMapper $formMapper)
                     'ROLE_ADMIN_TON_MACRON' => 'ROLE_ADMIN_TON_MACRON',
                     'ROLE_ADMIN_LEGISLATIVES' => 'ROLE_ADMIN_LEGISLATIVES',
                     'ROLE_ADMIN_ADHERENTS' => 'ROLE_ADMIN_ADHERENTS',
+                    'ROLE_ADMIN_SUMMARY' => 'ROLE_ADMIN_SUMMARY',
```

Comment:
This array is ugly, to simplify this kind of stuff just remove the keys and add:
```php
`choice_label` => function ($choice) { return $choice; },
```